apps/drupal-default/particle_theme/templates/form/form-element-label.html.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * Theme override for a form element label.
+ *
+ * Available variables:
+ * - title: The label's text.
+ * - title_display: Elements title_display setting.
+ * - required: An indicator for whether the associated form element is required.
+ * - attributes: A list of HTML attributes for the label.
+ *
+ * @see template_preprocess_form_element_label()
+ */
+#}
+{%
+  set classes = [
+    'text-sm',
+    'pb-1',
+    'tracking-wider',
+    'font-semibold',
```

Comment:
This will modify the form labels across the site, but very minorly. I looked at a few forms in the library section that I knew were there. LMK if there are other specific ones I should check.